### DISCUSSION TITLE
Is there any way to edit the webpage which pops up after connecting to wifi?

### DISCUSSION THREAD

**Discussion by josephcherayil** [HomeSpan/HomeSpan#551] [General]
Hi,
Brilliant library i am loving it, i have made a switch board which is connected to a fan, light and charger and its rock solid since one month.
I was exploring the ways to connect to wifi, resetting etc without connecting to a desktop. Is there any way we can edit the web page which pops up after connecting to wifi "homespan setup" ?

**Comment by HomeSpan** [comment]
Glad you are enjoying the library.  At present there is no way to edit the HomeSpan setup page.  However, you can create a separate web server and host your own web pages that can be accessed any time.  See the [Programmable Hub](https://github.com/HomeSpan/HomeSpan/tree/master/examples/Other%20Examples/ProgrammableHub) example for how to host a separate web server.
